use-cases/vehicle-routing/src/main/java/org/acme/vehiclerouting/domain/geo/HaversineDrivingTimeCalculator.java

```diff
@@ -0,0 +1,56 @@
+package org.acme.vehiclerouting.domain.geo;
+
+import org.acme.vehiclerouting.domain.Location;
+
+/**
+ * Calculates the driving time (in seconds) between two locations by calculating their Haversine distance in meters
+ * assuming average speed {@link #AVERAGE_SPEED_KMPH}.
+ */
+public class HaversineDrivingTimeCalculator implements DrivingTimeCalculator {
```

Comment:
Since this class has no instance fields, maybe we make it a singleton?

Reply:
OK, done.